### ISSUE TITLE
How to get AUC

### ISSUE THREAD

**Issue by ZDDWLIG** [GraphPKU/NeuralCommonNeighbor#6]
Thank you for your excellent work! I want to use AUC metrics on my own dataset for evaluation. But I can't find the logits of the predictor output. Could you please provides some details on how to use AUC for evaluation?

**Comment by Xi-yuanWang** [comment]
If you want to get test AUC score, you can add the following code at line 136 at `NeighborOverlap.py`. 

```
pos_test_pred = pos_test_pred.flatten()
neg_test_pred = neg_test_pred.flatten()
ypred = torch.cat((pos_test_pred, neg_test_pred), dim=0).cpu().numpy()
ytrue =  torch.cat((torch.ones_like(pos_test_pred), torch.zeros_like(neg_test_pred)), dim=0).cpu().numpy()
from sklearn import metrics
aucscore = metrics.roc_auc_score(ytrue, ypred)
``` 

`pos_test_pred` and `neg_test_pred` are logits for positive and negative links respectively.
